src/components/MessageListener/MessageListener.js

```diff
@@ -7,7 +7,8 @@ function MessageListener() {
   );
 
   useEffect(() => {
-    const unlistens = [];
+    const abrtCtrl = new AbortController();
```

Comment:
Does it work if you use ? 
` const { signal, abort } = new Abort Controller()`

It could be more readable